package.json

```diff
@@ -78,49 +71,43 @@
 		"build-server-stats": "NODE_ENV=production CALYPSO_ENV=production EMIT_STATS=true yarn run build-server",
 		"build-client": "yarn run build-client-evergreen",
-		"build-client-do": "webpack --config client/webpack.config.js --stats-preset errors-only $WEBPACK_OPTIONS",
-		"build-client-fallback": "BROWSERSLIST_ENV=defaults yarn run build-client-do",
-		"postbuild-client-fallback": "yarn run validate-fallback-es5",
+		"build-client-do": "webpack --config client/webpack.config.js --stats-preset errors-only",
+		"build-client-fallback": "BROWSERSLIST_ENV=defaults yarn run build-client-do && yarn run validate-fallback-es5",
 		"build-client-evergreen": "BROWSERSLIST_ENV=evergreen yarn run build-client-do",
 		"build-client-both": "CONCURRENT_BUILDS=2 concurrently -c cyan -n \"fallback ,evergreen\" \"yarn run build-client-fallback\" \"yarn run build-client-evergreen\"",
 		"build-client-if-prod": "node -e \"process.env.CALYPSO_ENV === 'production' && process.exit(1)\" || yarn run build-client-both && yarn run build-languages-if-enabled",
 		"build-client-if-desktop": "node -e \"(process.env.CALYPSO_ENV === 'desktop' || process.env.CALYPSO_ENV === 'desktop-development') && process.exit(1)\" || yarn run build-client-fallback",
 		"build-client-stats": "NODE_ENV=production CALYPSO_ENV=production EMIT_STATS=true yarn run build-client-evergreen",
-		"build-packages": "{ [ \"$SKIP_TSC\" = \"true\" ] || tsc --build packages/tsconfig.json; } && lerna run prepare --stream",
+		"build-packages": "{ [ \"${SKIP_TSC:-}\" = \"true\" ] || tsc --build packages/tsconfig.json; } && yarn workspaces foreach --all --parallel --verbose run prepare",
 		"build-languages": "yarn run translate && node bin/build-languages.js",
 		"build-languages-if-enabled": "node -e \"(process.env.BUILD_TRANSLATION_CHUNKS === 'true' || process.env.ENABLE_FEATURES === 'use-translation-chunks') && process.exit(1)\" || yarn run build-languages",
 		"calypso-doctor": "./node_modules/.bin/calypso-doctor",
 		"clean": "yarn run clean:packages && yarn run clean:build && yarn run clean:public && yarn run clean:translations",
-		"clean:build": "npx rimraf build client/server/bundler/*.json client/server/devdocs/search-index.js",
-		"clean:packages": "lerna run clean --stream",
+		"clean:build": "npx rimraf build 'client/server/bundler/*.json' client/server/devdocs/search-index.js",
+		"clean:packages": "yarn workspaces foreach --all --parallel --verbose --exclude 'wp-calypso' run clean",
 		"clean:public": "npx rimraf public",
-		"clean:translations": "npx rimraf build/strings calypso-strings.pot chunks-map.*.json",
-		"distclean": "yarn run clean && npx rimraf node_modules client/node_modules desktop/node_modules apps/*/node_modules packages/*/node_modules test/e2e/node_modules .cache apps/*/.cache vendor",
+		"clean:translations": "npx ri mraf build/strings calypso-strings.pot chunks-map.*.json",
```

Comment:
It doesn't seem to affect the running (somehow??), but I think this should be `rimraf` here!

Reply:
Good catch!